romancal/datamodels/schemas/reference_files/readnoise.schema.yaml

```diff
@@ -0,0 +1,12 @@
+%YAML 1.1
+---
+$schema: http://stsci.edu/schemas/asdf/asdf-schema-1.0.0
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/readnoise.schema"
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_gainfact.schema
+- type: object
+  properties:
+    data:
```

Comment:
See comment [here](https://github.com/spacetelescope/romancal/pull/114/files#r577590491), I think the validators dropped here (`datatype`, `ndim`, `default`) are valuable, and recommend `tag:` instead of `$ref`.

If you do opt to keep `$ref`, the relative URI here is incorrect.  It's evaluated against the `id` of the current schema:

`http://stsci.edu/schemas/roman_datamodel/reference_files/../core/asdf/ndarray-1.0.0`

which resolves to this:

`http://stsci.edu/schemas/roman_datamodel/core/asdf/ndarray-1.0.0`

The real ndarray-1.0.0 schema is not nested in "roman_datamodel".